### PR TITLE
Task/http commands

### PR DESCRIPTION
**Pull request by dmoranj**
Implements #11

ADD support for Push Commands HTTP.

### CODE REVIEW

File: lib/bindings/HTTPBindings.js [hunk 1]
```diff
@@ -133,7 +136,98 @@ function handleIncomingMeasure(req, res, next) {
...
+        };
+    }
+
+    options = {
+        url: device.endpoint,
+        method: 'POST',
+        body: ulParser.createCommandPayload(device, cmdName, cmdAttributes),
+        headers: {
+            'fiware-service': 'smartGondor',
+            'fiware-servicepath': '/gardens'
```
Comment (anabelengp) [inline lib/bindings/HTTPBindings.js]: default values

File: lib/bindings/HTTPBindings.js [hunk 2]
```diff
@@ -133,7 +136,98 @@ function handleIncomingMeasure(req, res, next) {
...
+                updateCommand(apiKey, device, body, callback);
+            }
+        });
+    };
+}
+
+/**
+ * Handles a command execution request coming from the Context Broker. This handler should:
+ *  - Identify the device affected by the command.
+ *  - Send the command to the appropriate MQTT topic.
```
Comment (anabelengp) [inline lib/bindings/HTTPBindings.js]: wrong comment (binding)

File: lib/bindings/HTTPBindings.js (hunk 1 above)
Comment (dmoranj) [inline lib/bindings/HTTPBindings.js]: Fixed in 0396e6f

File: lib/bindings/HTTPBindings.js (hunk 2 above)
Comment (dmoranj) [inline lib/bindings/HTTPBindings.js]: Fixed in 0396e6f

**Comment by anabelengp** [comment]
LGTM

**Pull request merged**